subcommand/controller/command.go

```diff
@@ -0,0 +1,117 @@
+package controller
+
+import (
+	"context"
+	"flag"
+	"os"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/hashicorp/consul-ecs/awsutil"
+	"github.com/hashicorp/consul-ecs/controller"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/go-hclog"
+	"github.com/mitchellh/cli"
+)
+
+const (
+	flagAgentSecretARN   = "agent-secret-arn"
+	flagSecretNamePrefix = "secret-name-prefix"
+
+	consulCACertEnvVar = "CONSUL_CACERT_PEM"
```

Comment:
is it a pem because we're injecting via env var and you don't want to write it to file?

Reply:
yeah exactly. Also, if you try to re-use the `CONSUL_CACERT` env var which Consul thinks points to a file, everything breaks even you overwrite values returned by `DefaultConfig()` because `api.NewClient` calls `defaultConfig` again (speaking from experience and a lot of printf debugging 😄 ).

Reply:
What if the command in the container first wrote it to file before executing `consul-ecs acl-controller`?

Reply:
It's also doable, although it would require us to overwrite the entry point of the container which we currently are not doing. I didn't want to mess with that. I could see though that there advantages and disadvantages to either solution.

Reply:
I think for consistency keeping `CONSUL_CACERT` across things would be better but don't feel too strongly.